api-scanner/src/test/resources/kotlin-auto-generated-class/kotlin/net/corda/example/ContainsAutoGeneratedClass.kt

```diff
@@ -0,0 +1,19 @@
+package net.corda.example
+
+class ContainsAutoGeneratedClass {
+    private open class PrivateInnerClass
+
+    private inline fun runSomething(crossinline action: PrivateInnerClass.() -> Unit): PrivateInnerClass {
+        return object: PrivateInnerClass() {
+            fun run() = action()
+        }
+    }
+
+    private val timer = PrivateInnerClass()
+
+    fun methodContainingInlinedFunction() {
+        runSomething {
+            System.out.println()
```

Comment:
As a Kotlin example, this should probably be `println()` rather than `System.out.println()`. But that's just me being _really_ picky :smiling_imp:.

Reply:
Like my mum always says, if a job's worth doing, it's worth doing right.